spec/index.html

```diff
@@ -7882,23 +7882,16 @@ <h2>Definition of SPARQL</h2>
         abstract query is then evaluated on an RDF dataset.</p>
       <section id="initDefinitions">
         <h3>Initial Definitions</h3>
```

Comment:
Suggestion for a different way to organise the material.

Replace the definition box for "RDF Dataset" with a paragraph that says the definition comes from RDF concepts. 

As it is important, repeat the definition written in the SPARQL style but not as a definition box. The SPARQL form is used in dataset merge that follows.

Active graph and merge can be in the same section as currently.
This leaves the numbering the same.

Reply:
Like Andy, I would also prefer to keep the subsection called "RDF Dataset".

Reply:
> Replace the definition box for "RDF Dataset" with a paragraph that says the definition comes from RDF concepts.
>
> As it is important, repeat the definition written in the SPARQL style but not as a definition box. The SPARQL form is used in dataset merge that follows.

@rubensworks To achieve the latter you can do the following thing. After the brief paragraph that Andy suggests to have first, you can introduce the more formal definition by another paragraph (_not_ in a definition box) that begins with:

For the following definitions, we capture each RDF dataset as a set: ...

Reply:
Does it make sense for the example input to have separate boxes for the default and named graphs? At the time, TriG didn't exist (at least in a standardized form), so maybe, for Example  1 in [Examples of RDF Datasets](https://pr-preview.s3.amazonaws.com/w3c/sparql-query/152/ca14da9...09ba854.html#exampleDatasets) could be the following:

```turtle
PREFIX dc: <http://purl.org/dc/elements/1.1/>
PREFIX foaf: <http://xmlns.com/foaf/0.1/>

<http://example.org/bob>    dc:publisher  "Bob" .
<http://example.org/alice>  dc:publisher  "Alice" .

GRAPH <http://example.org/bob> {
  _:a foaf:name "Bob" .
  _:a foaf:mbox <mailto:bob@oldcorp.example.org> .
}

GRAPH <http://example.org/alice> {
  _:b foaf:name "Alice" .
  _:b foaf:mbox <mailto:alice@work.example.org> .
}
```